formats/external_content.jsonnet

```diff
@@ -0,0 +1,32 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: "external_content",
+  base_path: "forbidden",
+  routes: "forbidden",
+  description: "required",
+  rendering_app: "forbidden",
+  edition_links: {},
+  links: {},
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      required: [
+        "url",
+      ],
+      properties: {
+        url: {
+          description: "A URL for the external resource",
+          type: "string",
+          format: "uri",
+        },
+        comment: {
+          description: "An internal description of why the link is relevant to GOV.UK users",
+          type: "string",
```

Comment:
I think I would like to see comment be a required field so that it can't be skipped.

Reply:
@MatMoore what's your opinion on this?

Comments are currently optional in search-admin, but it looks like every link has one anyway. (Though they're mostly automated comments which link to https://github.com/gds-attic/recommended-links)

Reply:
Having discussed this with @MatMoore, we're going to remove this field from the schema entirely. We'll continue to store it in the search-admin DB but it's an internal note so it doesn't currently make sense to send it to the publishing API.